avocado/utils/software_manager/backends/yum.py

```diff
@@ -129,41 +156,58 @@ def add_repo(self, url):
             self.repo_config_parser.add_section(section_name)
             self.repo_config_parser.set(section_name, 'name',
                                         'Avocado managed repository')
-            self.repo_config_parser.set(section_name, 'url', url)
+            self.repo_config_parser.set(section_name, 'baseurl', url)
```

Comment:
`baseurl` worked for me when testing, not `url`. Couldn't find any info on the difference between the two, if someone could enlighten me it would be much appreciated :)